Update Linux-Test-Project-Tests.sh

chvalean opened the pull request:
On a minimal OS an input might appear during LTP run:
Checking for required user/group ids
If any required user ids and/or groups are missing, would you like these created? [y/N]

Other structural changes to the code to make use of newer functions.

nicepat commented on JenkinsPipelines/Scripts/ReceivePartnerFiles.ps1:
```diff
@@ -111,22 +101,14 @@ if ($env:Kernel -eq "custom")
             $out = Start-BitsTransfer  -Source "$env:customKernelURL"
             if ($?)
             {
-                if ($env:customKernelURL.EndsWith(".deb"))
```
Is it ok with any file extension?

nicepat commented on JenkinsPipelines/Scripts/ReceivePartnerFiles.ps1:
```diff
@@ -160,8 +143,10 @@ if ($env:CustomVHD)
 
-        LogMsg "Copying '$env:CustomVHD' --> $PartnerUsernameShareDirectory\$BuildNumber-$env:CustomVHD"
-        Move-Item CustomVHD $PartnerUsernameShareDirectory\$BuildNumber-$env:CustomVHD -Force
+        #LogMsg "Copying '$env:CustomVHD' --> $PartnerUsernameShareDirectory\$BuildNumber-$env:CustomVHD"
```
remove comment

nicepat commented on XML/TestCases/SmokeTests.xml:
```diff
@@ -12,9 +12,9 @@
-		<testScript>BVT-VERIFY-HOSTNAME.py</testScript>
-		<PowershellScript>BVT-VERIFY-HOSTNAME.ps1</PowershellScript>
-		<files>.\Testscripts\Linux\nslookup.py,.\Testscripts\Linux\BVT-VERIFY-HOSTNAME.py,.\Testscripts\Linux\azuremodules.py</files>
+		<testScript>WALA-VERIFY-HOSTNAME.py</testScript>
```
'tag' should change to 'wala'?

iamshital replied:
added tag wala

chvalean committed:
Update Linux-Test-Project-Tests.sh

On a minimal OS an input might appear during LTP run:
Checking for required user/group ids
If any required user ids and/or groups are missing, would you like these created? [y/N]

Other structural changes to the code to make use of newer functions.
